Examples/Image/Detection/FastRCNN/BrainScript/PARAMETERS.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, datasetName):
         self.cntk_max_epochs = -1                # set per data set below
         self.cntk_momentum_time_constant = -1    # set per data set below
 
+        # for Distributed learner
+        self.num_quantization_bits = 1   # set for distributed learner 
```

Comment:
I'd start from 32-bit data parallel SGD for accuracy, unless communication cost is higher than computation.

Reply:
Are there any reference as to distributed learning? I didn't find the detailed specification on it.